tests/integration_tests/core_tests.py

```diff
@@ -1013,7 +1013,6 @@ def test_tabstate_update(self):
         data = {"sql": json.dumps("select 1"), "latest_query_id": json.dumps(client_id)}
         response = self.client.put(f"/tabstateview/{tab_state_id}", data=data)
         assert response.status_code == 400
-        assert response.json["error"] == "Bad request"
```

Comment:
Validating the `status_code` here is sufficient as now the error will contain database specific exceptions.